src/display/editor/editor.js

```diff
@@ -124,9 +128,17 @@ class AnnotationEditor {
 
   /**
    * Initialize the l10n stuff for this type of editor.
-   * @param {Object} _l10n
+   * @param {Object} l10n
    */
-  static initialize(_l10n) {
+  static initialize(l10n, options = null) {
+    AnnotationEditor._l10nPromise ||= new Map([
+      ["alt_text_button_label", l10n.get("alt_text_button_label")],
```

Comment:
If we ever want to add more strings here, should we perhaps just use the following format instead to reduce duplication?
```suggestion
      ["alt_text_button_label"].map(str => [str, l10n.get(str)])
```